search/search_repository.go

```diff
@@ -735,7 +735,7 @@ func (r *GormSearchRepository) SearchFullText(ctx context.Context, rawSearchStri
 			spew.Dump(value)
 			return nil, 0, errors.NewInternalError(ctx, errs.Wrap(err, "failed to load work item type"))
 		}
-		wiModel, err := wiType.ConvertWorkItemStorageToModel(value)
+		wiModel, err := workitem.ConvertWorkItemStorageToModel(wiType, &value)
```

Comment:
Hm, I didn't know we had two functions doing almost the same thing. Is the old one used anywhere or should we remove it?

Reply:
Actually the old is used from new one. Probably we can get rid of the old one.

Reply:
Removed. 857f0bc0a0302d792195841e59f58967046d38d3